Nutraceutical Additions (#314)
greenelab/covid19-review: content/20.treatments.md

#### Nutraceuticals

Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].

> The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but they must also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:https://doi.org/10.1208/ps050325].

Reviewer: ```suggestion
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but they must also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
```

Several mechanisms for SPM have been proposed, including preventing the release of pro-inflammatory cytokines and chemokines or increasing phagocytosis of cellular debris by macrophages [@doi:10.1038/nature13479].  
In influenza, SPM promote anti-viral B lymphocytic activities [@doi:10.4049/jimmunol.1302795], and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
It is hypothesised that SPM may aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 [@doi:10.1007/s10555-020-09889-4].
However, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027].
The effectiveness of n-3 PUFA against infections would be dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].

One pathological aspect of COVID-19 infection that was late to be recognized is the increased risk of thrombotic complications [@doi:10.1016/j.thromres.2020.04.013]. 

> Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005;@doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 

Reviewer 2: ```suggestion
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 
```

SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses [@doi:10.1038/nri.2015.4].
Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
SPM potentially function through various mechanisms, which includes preventing the release of pro-inflammatory cytokines and chemokines and phagocytosing of cellular debris by macrophages [@doi: 10.1038/nature13479].  
In influenza, SPM promote anti-viral B lymphocytic activities [@doi:10.4049/jimmunol.1302795] and protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
It is hypothesised that SPM may aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 [@doi:10.1007/s10555-020-09889-4].
Moreover, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027].

> The effectiveness of n-3 PUFA against infections is dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].

Author: The literature in relation to n-3 fatty acids is not there yet for infection specifically. Hence my phrasing not being particularly definitive I guess. However, they are valued for their anti-inflammatory properties. In any case, evidence shows that the dosage and timing  are important but there is no clear consensus on that for several reasons. I will review again.

One pathological aspect of COVID-19 infection that was late to be recognised  was the increased risk of thrombotic complications [@doi:10.1016/j.thromres.2020.04.013]. 
Considering, there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005;@doi:10.1016/j.blre.2020.100694] they may have therapeutic value against the prothrombotic complications of COVID-19. 

> This may be particularly important since patients receiving pharmaceutical antiplatelet therapies may become infected.

Author: I have made this section clearer. That was my bad, see if it makes more sense now :)

Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; p = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%). 
Evidence of sexual dimorphism was apparent, as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; p = 0.0006). 
Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (p = 0.001).   
Despite these two observational studies potentially linking vitamin D with COVID-19, an examination of the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050]. 
This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.

> One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].

Reviewer 2: ```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```
```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url:https://clinicaltrials.gov/ct2/show/NCT04334005].
```

###### Current Evidence

A recent review postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.

> This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.3390/nu12040988].

Reviewer: ```suggestion
This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low [@doi:10.3390/nu12040988].
```

The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.

> While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.  

Reviewer: ```suggestion
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
```

Author: effect/affect deadly sin by me!!!

> Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.

Reviewer: ```suggestion
Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
```

These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 

> As the pandemic evolves, research has started to interrogate many of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 in terms of its prophylactic and therapeutic value. 

Reviewer: ```suggestion
As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 and sought to shed light on whether there is any  prophylactic and/or therapeutic relationship. 
```

> A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than that of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), which was also confirmed when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 

Reviewer: ```suggestion
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), an association that held when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 
```

> These findings seem to be supported by another observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia in Belgium, whereby 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalisation [@doi:10.1101/2020.05.01.20079376]. 

Reviewer: ```suggestion
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376]. 
```

> A significant difference of 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.

Reviewer: ```suggestion
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
```

Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; p = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%). 

> Evidence of sexual dimorphism was reported as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were particularly deficient in 25-hydroxyvitamin D versus the controls (67% versus 49%; p = 0.0006). 

Reviewer: ```suggestion
Evidence of sexual dimorphism was apparent, as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; p = 0.0006). 
```

Notably, vitamin D deficiency was progressively lower in males with advancing radiological disease stages (p = 0.001).   
Despite these two observational studies potentially linking vitamin D with COVID-19, an examination of the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050]. 
This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.

> One trial is currently investigating the utility of vitamin D as an immune modulating agent by monitoring  if there is an improvement of health status in non-severe symptomatic patients infected with COVID-19 as well as determining if vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].

Reviewer: ```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```

###### Current Evidence

> A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19.

Reviewer 3: I would stick to the Calcifediol/Calcitriol or 25-OH/1,25-OH terminology and introduce them here because they are short and comprehensible. Where is the preprint reference?

This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low [@doi:10.3390/nu12040988].
The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  

> Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.

Reviewer 3: From my understanding, a seasonal disease is something different than a disease with seasonal severity. What exactly is the assumption of this review?

Author: This line refers to seasonal incidence (more infection around January etc.), in line with the hypothesis that vitamin D is a potential factor.
From my understanding, there has been little discussion on potential seasonal severity (the human impact of the virus e.g. increased hospitalisation levels etc.); Although, viral mutation patterns are being continually monitored.

These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 

> As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 and sought to shed light on whether there is any  prophylactic and/or therapeutic relationship. 

Reviewer 3: Is this the same review? If so, the reference and authors might better be mentioned in the first sentence.

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections.
Nevertheless, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@url:https://www.preprints.org/manuscript/202003.0199/v1].
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.

> As a result, the administration of zinc, vitamin C, and vitamin D in conjunction with hydroxychloroquine are being investigated for their prophylactic effects in healthcare workers and their first degree relatives in Turkey [@url:https://clinicaltrials.gov/ct2/show/NCT04326725].

Reviewer: ```suggestion
As a result, the administration of zinc, vitamin C, and vitamin D in conjunction with hydroxychloroquine are being investigated for their prophylactic effects in healthcare workers and their first-degree relatives in Turkey [@url:https://clinicaltrials.gov/ct2/show/NCT04326725].
```
